atlasdb-dagger/src/main/java/com/palantir/atlasdb/services/LockAndTimestampModule.java

```diff
@@ -36,7 +36,8 @@
                 config.atlasDbConfig(),
                 resource -> { },
                 LockServiceImpl::create,
-                () -> config.atlasDbSupplier().getTimestampService());
+                () -> config.atlasDbSupplier().getTimestampService(),
+                null); //TODO
```

Comment:
@jeremyk-91 @fsamuel-bs can someone help me figure out what to do here?

Reply:
We will probably want to create a `@Qualifier` annotation for the refreshing scheduled executor and add that as a param to this method, and provide a default implementation

Reply:
The cheap solution is to hardcode a default...

Better solution is to have a qualifier annotation for the refreshing scheduled executor as @schlosna mentions.